cabby/data/extract.py

```diff
@@ -101,3 +101,31 @@ def get_data_by_region(region: Text) -> Sequence:
 
     return get_wikigeo_data(wikidata_items)
 
+
+def split_dataset(dataset, percentage_train:float, percentage_dev:float):
+    '''Splits the dataset into train-set, dev-set, test-set according to the ref_qid" 
+    Arguments: 
+      percentage_train(float in [0,1]): percentage of the train-set. percentage_dev(float in [0,1]): percentage of the dev-set. 
```

Comment:
Line is >80 chars

Reply:
Doesn't look done -- did you forget to push this change?

Reply:
I just pulled the code and it looks different - two line and not one.

Reply:
Looks good in committed code now.